chsdi/models/vector/stopo.py

```diff
@@ -335,13 +376,15 @@ class Landeskarte50zusMetadata (Base, ShopProductGroupClass, Vector):
     __table_args__ = ({'schema': 'datenstand', 'autoload': False})
     __tablename__ = 'view_gridstand_lk50zus_shop'
     __bodId__ = 'ch.swisstopo.lk50-papierkarte-zusammensetzung.metadata'
+    price = Column('price', Integer)
```

Comment:
Sorry I see that SegelFlug300Digital doesn't have a price and is part of ShopProductClass